### PR TITLE
Feature/timeline

### PR DESCRIPTION
**Pull request by RRikken**
This is the first iteration of the timeline component. It only uses the Sloth and Team models at the moment.

### CODE REVIEW

File: heroeslounge/models/Sloth.php [hunk 1]
```diff
@@ -38,22 +39,73 @@ class Sloth extends Model
...
+	    }
+    }
+
+    public function afterDelete()
+    {
+	    $this->_saveTimelineEntry('Sloth.Deleted');
     }
-}
 
+    private function _saveTimelineEntry($entryType)
```
Comment (Astraeus-) [inline heroeslounge/models/Sloth.php]: Perhaps turn the collection of if-statements for the $entryType into a switch statement?

Comment (RRikken) [inline heroeslounge/models/Sloth.php]: This function could use a little cleanup.

Comment (RRikken) [inline heroeslounge/models/Sloth.php]: True, I should use a fallthrough on the last three options right?

Comment (Astraeus-) [inline heroeslounge/models/Sloth.php]: Yes, those 3 cases would  want to be processed the same way.

**Pull request merged**